components/prism-psl.js

```diff
@@ -0,0 +1,28 @@
+Prism.languages.psl = {
+	'comment': [
+		{
+			pattern: /#.*/,
+			greedy: true
+		}
+	],
```

Comment:
No need for the array. You can just inline the object literal.
```suggestion
	'comment': {
		pattern: /#.*/,
		greedy: true
	},
```